app/utils.py

```diff
@@ -202,3 +203,13 @@ def appstream2dict(reponame: str):
         apps[appid] = app
 
     return apps
+
+
+def get_appids(path):
+    try:
+        with open(
+            path,
+        ) as file_:
+            return json.load(file_)
+    except IOError:
```

Comment:
Is there any value in logging the IOError here?  In case things stop working.

Reply:
Possibly? I didn't write this, just moved it from picks.py.